app/server/appsmith-plugins/firestorePlugin/src/main/java/com/external/plugins/FirestorePlugin.java

```diff
@@ -952,12 +952,12 @@ public Mono<Firestore> datasourceCreate(DatasourceConfiguration datasourceConfig
         @Override
         public Mono<DatasourceTestResult> testDatasource(DatasourceConfiguration datasourceConfiguration) {
             String printMessage = Thread.currentThread().getName() + ": testDatasource() called for Firestore plugin.";
-            System.out.println(printMessage);
+            log.debug(printMessage);
             return datasourceCreate(datasourceConfiguration).flatMap(connection -> {
                 try {
                     connection.listCollections();
                 } catch (FirestoreException e) {
-                    System.out.println("Invalid datasource configuration: " + e.getMessage());
+                    log.debug("Invalid datasource configuration: " + e.getMessage());
```

Comment:
```suggestion
                    log.error("Invalid datasource configuration: " + e.getMessage());
```

Please user log.error when dealing with exception.